### ISSUE TITLE
Missing tsconfig.schema.json for volar-0.30.5

### ISSUE THREAD

**Issue by NoahStahl** [johnsoncodehk/volar#869]
I'm seeing the following message when viewing `tsconfig.json` in my project. This appeared today, I suspect as a result of Volar update? Also updated `vue-tsc` to 0.30.5 but rolling back to 0.30.2 doesn't resolve.

```
Problems loading reference 'file:///c%3A/Users/Noah/.vscode/extensions/johnsoncodehk.volar-0.30.5/schemas/tsconfig.schema.json': Unable to load schema from 'c:\Users\Noah\.vscode\extensions\johnsoncodehk.volar-0.30.5\schemas\tsconfig.schema.json': ENOENT: no such file or directory, open 'c:\Users\Noah\.vscode\extensions\johnsoncodehk.volar-0.30.5\schemas\tsconfig.schema.json'
```
The current contents of the `schemas` directory:

![image](https://user-images.githubusercontent.com/7528994/149782019-35f27258-afa0-4023-a242-6826e795f804.png)

**Comment by johnsoncodehk** [comment]
Thanks for the report!

**Comment by crystalfp** [comment]
This file is not present in the same place also for version 0.30.4

**Comment by yaegassy** [comment]
@crystalfp I believe this issue will be resolved in the next release (v0.30.6).
